frontend/src/pages/Login.tsx

```diff
@@ -60,9 +66,10 @@ export const Login: React.FC = (): JSX.Element => {
               {error && <FormErrorMessage>Incorrect email or password.</FormErrorMessage>}
             </FormControl>
 
-            <Button colorScheme="teal" type="submit" width="full" mt={4} disabled={loading}>
+            <Button colorScheme="teal" type="submit" width="full" mt={4} disabled={!verified}>
               Submit
             </Button>
+            <ReCAPTCHA sitekey="6LdSRdAfAAAAAE5Qbv-Qg6vmiATbXK8SPJFDu9Ai" onChange={handleVerification} />
```

Comment:
Also add `Center` to imports from Chakra at the top
```suggestion
            <Center pt="50px">
              <ReCAPTCHA sitekey="6LdSRdAfAAAAAE5Qbv-Qg6vmiATbXK8SPJFDu9Ai" onChange={handleVerification} />
            </Center>
```